src/components/Comment/docs.md

```diff
@@ -0,0 +1,83 @@
+### `<Comment />`
+
+A `<Comment />` is a combination of `<CommentHeader />` and the comment content. The actions are not part of this component.
+
+```react|noSource,span-4
+<Comment
+  timeago='2h'
+  displayAuthor={{
+    profilePicture: '/static/profilePicture.png',
+    name: 'Paul Ullrich',
+    credential: {description: 'Bundesrat', verified: true}
+  }}
+  score={8}
+  content="This has to be the first time that an author who has written a Fantasy novel (The Buried Giant), is awarded with the Nobel Prize. He has also writted Sci-fi (Never Let Me Go), but Doris Lessing was first there (Shikasta). Wow, I did not expect this one. Don't get me wrong, I *love* his books, but he's way more mainstream than I expected."
```

Comment:
I'm not a lawyer, but just wanted to raise the question whether we should copy actual reddit comments or better use some Lorem Ipsum or self-created text?